src/sidebar/components/group-list-item.js

```diff
@@ -25,6 +25,7 @@ function GroupListItem({
   const activityUrl = group.links.html;
   const hasActionMenu = activityUrl || group.canLeave;
   const isSelectable = !group.scopes.enforced || group.isScopedToUri;
+  const isSubmenuEmpty = !activityUrl && !group.canLeave && isSelectable;
```

Comment:
There is already a `hasActionMenu` variable here which can be used. Note that whether or not a group is selectable doesn't affect whether it has menu items. A group might be unselectable because it is restricted to certain URLs, but the menu item to go to its activity page is still available. The tests on this PR pass with the following diff:

```diff
diff --git a/src/sidebar/components/group-list-item.js b/src/sidebar/components/group-list-item.js
index 63c32d621..4744a7644 100644
--- a/src/sidebar/components/group-list-item.js
+++ b/src/sidebar/components/group-list-item.js
@@ -25,7 +25,6 @@ function GroupListItem({
   const activityUrl = group.links.html;
   const hasActionMenu = activityUrl || group.canLeave;
   const isSelectable = !group.scopes.enforced || group.isScopedToUri;
-  const isSubmenuEmpty = !activityUrl && !group.canLeave && isSelectable;
 
   const focusedGroupId = useStore(store => store.focusedGroupId());
   const isSelected = group.id === focusedGroupId;
@@ -84,7 +83,7 @@ function GroupListItem({
       isDisabled={!isSelectable}
       isExpanded={hasActionMenu ? isExpanded : false}
       isSelected={isSelected}
-      isSubmenuVisible={isSubmenuEmpty ? undefined : isExpanded}
+      isSubmenuVisible={hasActionMenu ? isExpanded : undefined}
       label={group.name}
       onClick={isSelectable ? focusGroup : toggleSubmenu}
       onToggleSubmenu={toggleSubmenu}
```

Re-reading this code though has be asking - why do we have two separate props for submenu visibility? It looks to me like the `isExpanded` prop is only used to add the `is-expanded` class if the menu item has an open submenu. That logic could be changed to add the `is-expanded` class if `isSubmenuVisible === true` instead.